libraries/libcore/inc/status.h

```diff
@@ -52,6 +52,7 @@ typedef enum {
   STATUS_CODE_OUT_OF_RANGE,
   STATUS_CODE_UNIMPLEMENTED,
   STATUS_CODE_UNINITIALIZED,
+  STATUS_CODE_CRC_MISMATCH,
```

Comment:
Hmm, I know of at least one other use case where CRC mismatch would be useful - I'm planning on incorporating CRCs into CAN ACKs. I'm fine with internal error or something similar, but I'm just putting that out there.